admin/formprocess.php

```diff
@@ -44,7 +44,7 @@
   }
 } else {
   $insert_comic_query = "INSERT INTO comics (series_id, issue_number, story_name, release_date, plot, cover_image, original_purchase, wiki_id)
-  VALUES ('$series_name', '$issue_number', '$story_name', '$release_date', '$plot', 'images/$cover_image_file', '$original_purchase', '$wiki_id')";
+  VALUES ('$series_id', '$issue_number', '$story_name', '$release_date', '$plot', 'images/$cover_image_file', '$original_purchase', '$wiki_id')";
```

Comment:
$series_name != $series_id. Nothing was being saved! Now I just have to match the series_id in the database to the name pulled from the Marvel Wikia (series_name form)